Updates operator docs (#2622)
VictoriaMetrics/VictoriaMetrics: docs/operator/high-availability.MD

 This configuration produces 5 deployments with 2 replicas at each. Each deployment has own shard num
and scrapes only 1/5 of all targets.

### StatefulMode

>  In `StatefulMode` `VMAgent` doesn't lose state for PersistentQueue after pod restart.

Reviewer: ```suggestion
 In `StatefulMode` `VMAgent` doesn't lose state of the PersistentQueue (file-based buffer size for unsent data) on pod restarts.
```

>  Operator creates `StatefulSet` for it, and with provided `PersistenVolumeClaimTemplate`  at `StatefulStorage` configuration param, metrics queue stored at disk.

Reviewer: ```suggestion
 Operator creates `StatefulSet` and, with provided `PersistenVolumeClaimTemplate` at `StatefulStorage` configuration param, metrics queue is stored on disk.
```
